test_utils/src/messages.rs

```diff
@@ -59,19 +64,68 @@ pub fn test_shared_object_transactions() -> Vec<Transaction> {
                 CallArg::Pure(16u64.to_le_bytes().to_vec()),
                 CallArg::Pure(bcs::to_bytes(&AccountAddress::from(sender)).unwrap()),
             ],
-            /* max_gas */ 10_000,
+            MAX_GAS,
         );
         let signature = Signature::new(&data, &keypair);
         transactions.push(Transaction::new(data, signature));
     }
     transactions
 }
 
-/// Make a test certificates for each shared-object transaction.
-pub async fn test_shared_object_certificates() -> Vec<CertifiedTransaction> {
+/// Make a transaction to publish a test move contracts package.
+pub fn publish_move_package_transaction(gas_object: Object) -> Transaction {
+    let build_config = BuildConfig::default();
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.push("../sui_programmability/examples/basics");
```

Comment:
Might want to make this an argument to make this function more generic.

Reply:
Good point https://github.com/MystenLabs/sui/issues/974